### PR TITLE
Updated Readme to disinguish from ST7735R driver

### LINKED ISSUE #4: Distinguish this library from ST7735R in the README
Twice in the past few days people have attempted to use this library for an ST7735R display. It might be helpful if the README mentions something like "if you have an ST7735R display, use this other library". And maybe symmetrically there.

### PR DESCRIPTION
**Pull request by makermelissa**
Fixes #4.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -18,11 +18,11 @@ displayio driver for ST7735B TFT-LCD displays.
 Hardware
 =========
 
-This driver does not work for any current Adafruit Displays and was created using initilization
-codes from the Arduino Library. If you have a newer Adafruit Display with a ST7735R or ST7735S chip, 
+This driver does not work for any current Adafruit Displays. It is intended to be used with displays
+with an ST7735B or similar chipset and was created using initilization codes from the Arduino Library.
```
Comment (dhalbert) [inline README.rst]: initilization -> initialization

**Review by dhalbert — CHANGES_REQUESTED** [review]
Just spelling!

File: README.rst (hunk 1 above)
Comment (makermelissa) [reply README.rst]: Good catch!

**Comment by makermelissa** [comment]
Fixed

**Review by dhalbert — APPROVED** [review]
Also thanks!

**Pull request merged**